src/eval.jl

```diff
@@ -54,7 +66,7 @@ handle("eval") do data
   lock(evallock)
   result = @dynamic let Media.input = Editor()
     withpath(path) do
-      @errs include_string(mod, text, path, line)
+      @errs include_string(mod, text, path, line) ends_with_semicolon(text)
```

Comment:
This is what I did in the beginning but then you would want to signal the renderer not only if there 
was a semi-colon but also if there was an error.

And then @errs needs to return not only the result(error message or value) but also whether it was an error. maybe the error message have a special type that can be checked .. then it can be added as a condition for rendering , this will also expose the whole logic in code:

so is there a special type for the error message?

regarding evaluating multi-line in the editor, it works just as before but suppresses the output if the last line ends with semi-colon